crates/ironrdp-cliprdr-native/src/windows/clipboard_impl.rs

```diff
@@ -320,7 +320,7 @@ pub(crate) unsafe extern "system" fn clipboard_subproc(
 
         // SAFETY: `data` is a valid pointer, returned by `Box::into_raw`, transferred to OS earlier
         // via `SetWindowSubclass` call.
-        let _ = unsafe { Box::from_raw(data as *mut WinClipboardImpl) };
+        let _unused = unsafe { Box::from_raw(data as *mut WinClipboardImpl) };
```

Comment:
I'm not sure about that, in all the modified lines in seems to me that immediately dropping the value is intended 🤔

Reply:
What I saw was that in all the cases, we ignored the `Result` from a method with `let _ =`. In this case, it's not required to drop it immediately.

Reply:
Make sense. Let me revert